backend/apollo/elections/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+
+class Answer(models.Model):
+    text = models.CharField(max_length=200)
+    votes = models.IntegerField()
+
+
+class Question(models.Model):
+    answers = models.ForeignKey(Answer, on_delete=models.CASCADE)
```

Comment:
This relationship is reversed - it gives one answer to multiple questions instead of one question to multiple answers. What we need is to put `question = models.ForeignKey...` inside the `Answer` model.

Reply:
Btw, to specify "answers" in this model, you might want to use `related_name`.